samcli/commands/bootstrap/command.py

```diff
@@ -22,9 +22,9 @@
 @pass_context
 @track_command
 def cli(ctx):
-    do_cli(ctx.region, ctx.profile)  # pragma: no cover
+    do_cli(ctx.region, ctx.profile, ctx.role_arn)  # pragma: no cover
```

Comment:
`role_arn` isn't apart of the context that I know of. You will need to add an `@option` annoation, you look at other commands to see how this is done.